static/js/components/email/EmailCompositionDialog.js

```diff
@@ -43,32 +44,22 @@ export default class EmailCompositionDialog extends React.Component {
     }
   };
 
-  automaticCheckbox = () => {
-    const {
-      activeEmail: { inputs },
-      updateEmailFieldEdit,
-    } = this.props;
-
-    return <Checkbox
-      label="Automatically send this message in the future, whenever new users join who meet these criteria"
-      className="email-automatic"
-      checked={inputs.sendAutomaticEmails || false}
-      value={true}
-      onCheck={e => {
-        updateEmailFieldEdit('sendAutomaticEmails', {
-          target: {
-            value: e.target.checked
-          }
-        });
-      }}
-    />;
-  };
+  renderAutomaticEmailSettings =
+    (sendAutomaticEmails: boolean, updateEmailFieldEdit: () => void): React$Element<*> => (
+    <AutomaticEmailOptions
+      automaticEmailType={sendAutomaticEmails ? EMAIL_CAMPAIGN : ONE_TIME_EMAIL}
```

Comment:
Why not just pass in a `sendAutomaticEmails` prop here? I don't see that the `automaticEmailType` is useful, we aren't likely to have any more than two different types (and if we do in the future we could add it in at that point). For now I think it would be better to just use a boolean throughout to represent the choice the user is making.